drivers/lora/sx12xx_common.h

```diff
@@ -11,6 +12,18 @@
 #include <drivers/lora.h>
 #include <device.h>
 
+int __sx12xx_configure_pin(struct device **dev, const char *controller,
+			   gpio_pin_t pin, gpio_flags_t flags);
```

Comment:
I think `#include <drivers/gpio.h>` should be in this header file, so `gpio_pin_t` and `gpio_flags_t` are known.

Reply:
> I think `#include <drivers/gpio.h>` should be in this header file, so `gpio_pin_t` and `gpio_flags_t` are known.

Ack

Reply:
Ugh, I seem to have missed this comment for some reason. I have updated the commit. Thanks for spotting it!